test/basic-querying.test.js

```diff
@@ -525,31 +525,31 @@ describe('basic-querying', function() {
 
     it('should find first record (default sort by id)', function(done) {
       User.all({ order: 'id' }, function(err, users) {
-        User.findOne(function(e, u) {
-          should.not.exist(e);
-          should.exist(u);
-          u.id.toString().should.equal(users[0].id.toString());
+        User.findOne(function(err, user) {
+          should.not.exist(err);
+          should.exist(user);
+          user.id.toString().should.equal(users[0].id.toString());
```

Comment:
I asked @yorkie to change it to `err/user` earlier because I didn't realize it was already named `e/u`. My bad! @yorkie Can you change it back to so these irrelevant lines aren't part of this commit?